config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -72,7 +72,7 @@ presubmits:
             - name: KUBERNETES_VERSION # CAPZ config
               value: "latest"
             - name: CLUSTER_TEMPLATE # CAPZ config
-              value: https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/release-1.12/templates/test/ci/cluster-template-prow-machine-pool-ci-version.yaml
+              value: https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/release-1.13/templates/test/ci/cluster-template-prow-machine-pool-ci-version.yaml
```

Comment:
I think you forget to update CAPZ version on L50 :) And there're several similar changes in the PR.

Reply:
thank you, fixed!